src/Common/Services/IQueryProxyService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Cmf.Foundation.BusinessObjects.QueryObject;
+using Cmf.Foundation.BusinessOrchestration.QueryManagement.OutputObjects;
+using System.Threading.Tasks;
+
+namespace Cmf.CustomerPortal.Sdk.Common.Services
+{
+    /// <summary>
+    /// Interface to be used as proxy in lbos request for queries
+    /// </summary>
+    public interface IQueryProxyService
```

Comment:
You forget to add this interface resolution to the QueryPorxyService to the CommonModule. Without that, the interface will not be injected correctly because the register will not solve the interface